pkg/cmd/docs-issue-generation/github.go

```diff
@@ -0,0 +1,294 @@
+// Copyright 2023 The Cockroach Authors.
+//
+// Use of this software is governed by the Business Source License
+// included in the file licenses/BSL.txt.
+//
+// As of the Change Date specified in that file, in accordance with
+// the Business Source License, use of this software will be governed
+// by the Apache License, Version 2.0, included in the file
+// licenses/APL.txt.
+
+package main
+
+import (
+	"fmt"
+	"strings"
+	"time"
+)
+
+func searchCockroachPRs(startTime time.Time, endTime time.Time) ([]cockroachPR, error) {
+	prCommitsToExclude, err := searchJiraDocsIssues(startTime)
+	if err != nil {
+		fmt.Println(err)
```

Comment:
Is ignoring the problem and continuing for sure what we want to do? If so, I would appreciate a comment to clarify that.

Reply:
That wasn't my original intention. I now return an error.